src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -70,8 +70,11 @@ static void HandleNodeResolve(void * context, DnssdService * result, const Span<
 
     nodeData.resolutionData.port = result->mPort;
 
+    ChipLogDetail(Discovery, "Node with instance name %s discovered. TXT:", nodeData.commissionData.instanceName);
```

Comment:
`DiscoveredNodeData` has `logDetail` method which is supposed to print that. Isn't it a duplicate?

Reply:
Yes, thank you!